### PR TITLE
Check renewal

### PR DESCRIPTION
**Pull request by makoto**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,28 +5,30 @@ This js library will allow wallets and dapps to remind users that they own expir
...
 const referrerAddress = '0x234...'
 
 const {
-  hasExpiringDomains,
   numExpiringDomains,
   renewalUrl
  } = await checkRenewal(userAddress, referrerAddress)
 
-if(hasExpiringDomains){
+if(hasExpiringDomains > 0){
```
Comment (jefflau) [inline README.md:19]: change to numExpiringDomains

File: README.md [hunk 2]
```diff
@@ -5,28 +5,30 @@ This js library will allow wallets and dapps to remind users that they own expir
...
 }
 ```
 
 - `renewalUrl` = `https:/app.ens.domains/address/0x123/renewal?referrer=0x345`
-- `numExpiringDomains` return number or `more than 100` if it has more than 100 names.
+- `numExpiringDomains` return only up to 100
+
+### Optional arguments
 
+-  `expirtyDate` = The default is set to 30 days from now
```
Comment (jefflau) [inline README.md:29]: expiryDate

File: example.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+const renewal  = require('./index');
+const userAddress ='0xA7F3659c53820346176f7E0E350780DF304db179'
+const referrerAddress ='0x5A384227B65FA093DEC03Ec34e111Db80A040615'
+renewal.checkRenewal(
+  userAddress,
+  referrerAddress,
+  {
+      expirtyDate:new Date("2020-05-04T16:10:38"),
```
Comment (jefflau) [inline example.js:8]: expiryDate

File: README.md (hunk 2 above)
Comment (jefflau) [reply README.md:29]: Also this isn't clear how this is filtering it. Maybe this should just be a number of days and you can work out the date from there

File: README.md [hunk 4]
```diff
@@ -5,28 +5,30 @@ This js library will allow wallets and dapps to remind users that they own expir
...
+-  `debug`       = The default is set to `false`. It displayes some debugging messaage if it's on.
 
 ## Notes
 
-- Any UI compoment will be offered as a seperate React component (eg: `@ensdomains/react-renew`).
```
Comment (jefflau) [inline README.md:32]: component

File: README.md (hunk 2 above)
Comment (makoto) [reply README.md:29]: expiryDate is just a date object, probably more flexible than setting number of dates as once can set the expiration time as well.

**Pull request merged**